bundles/org.openhab.binding.pulseaudio/src/main/java/org/openhab/binding/pulseaudio/internal/PulseAudioAudioSource.java

```diff
@@ -243,13 +248,13 @@ static class PulseAudioStream extends AudioStream {
         private final Logger logger = LoggerFactory.getLogger(PulseAudioAudioSource.class);
         private final AudioFormat format;
         private final InputStream input;
-        private final Consumer<Boolean> setIdle;
+        private final Runnable activity;
```

Comment:
Why is this activity required? I can't realize what it does.

Reply:
(see previous comment)